docs/edgeql/expressions/parameters.rst

```diff
@@ -0,0 +1,173 @@
+.. _ref_eql_expr_params:
+
+Parameters
+==========
+
+:edb-alt-title: Query Parameters
+
+A parameter reference is used to indicate a value that is supplied externally
+to an EdgeQL expression.
+
+An example of query, that can be parametrized:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT Person { birth_date } FILTER .first_name = 'John';
+    {
+        Object {birth_date: <cal::local_date>'1971-01-01'},
+    }
+
+The equivalent parametrized query:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT Person { birth_date } FILTER .first_name = <str>$name;
+    Parameter <str>$name: John
+    {
+        Object {birth_date: <cal::local_date>'1971-01-01'},
+    }
+
+Note how the actual parameter value is asked by REPL in a separate prompt,
+after the whole query is accepted.
+
+While parameters are useful in REPL, their power is mostly when using language
+bindings. Example of using parameters in Python:
```

Comment:
Hm, "their power is mostly" is incomplete. I'd replace this paragraph with something like "Here's how parameters can be used with language bindings, e.g. with Python:"

Reply:
Changed.